lua/satellite/handlers.lua

```diff
@@ -3,7 +3,7 @@
 ---@field name string
 ---@field ns integer
 ---@field init fun()
----@field update fun(bufnr: integer)
+---@field update fun(bufnr: integer,user_config: table)
```

Comment:
Config should be passed to `init()`

Reply:
sorry i couldn't figure out where to pass it

Reply:
It's ok. I'm still figuring a lot of this out. It still might change.